python/cuml/test/test_fil.py

```diff
@@ -246,15 +250,19 @@ def test_fil_skl_classification(n_rows, n_columns, n_estimators, max_depth,
                                            storage_type=storage_type)
     fil_preds = np.asarray(fm.predict(X_validation))
     fil_preds = np.reshape(fil_preds, np.shape(skl_preds_int))
-
-    fil_proba = np.asarray(fm.predict_proba(X_validation))
-    fil_proba = np.reshape(fil_proba, np.shape(skl_proba))
-
     fil_acc = accuracy_score(y_validation, fil_preds)
+    # fil_acc is within p99 error bars of skl_acc (diff == 0.017 +- 0.012)
```

Comment:
is p99 a typo?

Reply:
it's within 3 standard deviations, tens of runs for each test and more than a hundred aggregated.